airbyte-integrations/connectors/source-notion/source_notion/streams.py

```diff
@@ -158,7 +158,7 @@ def parse_response(self, response: requests.Response, stream_state: Mapping[str,
             state_lmd = stream_state.get(self.cursor_field, "")
             if isinstance(state_lmd, StateValueWrapper):
                 state_lmd = state_lmd.value
-            if not stream_state or record_lmd >= state_lmd:
+            if not stream_state or record_lmd > state_lmd:
```

Comment:
i think the `>=` is there to prevent accidentally losing data in the case of eventual consistency i.e: if at T1 you ask for data generated `<=` T1, then after T1 you ask for data generated `>=` T1, the second time you ask there might be some data generated exactly during T1 which wasn't returned the first time around

Honestly, I have no idea how common this is or if the Notion API is susceptible to this failure mode, but it's how all our API connectors tend to work. The deduplication guarantee is handled by the destination i.e: the destination should ensure that there is 1 record per primary key at the end of the day, and the source guarantees only an at-least-once delivery not an exactly-once delivery

Reply:
Gotcha, thanks for the clarification! I'll go ahead and close the PR